primitives/bridge/src/container_token_to_ethereum_message_exporter.rs

```diff
@@ -313,6 +319,16 @@ where
         }
     }
 
+    pub fn get_reanchored_location(&self, location: &Location) -> Result<Location, ()> {
+        let token_split = location.interior().clone().split_global().map_err(|_| ())?;
+
+        let container_token_from_tanssi = Location::new(0, token_split.1);
```

Comment:
Add a comment here explaining that this only works for tokens that belong to the container-chain (and not when they are rtying to export something different)